sdk/communication/Azure.Communication.CallAutomation/src/CallMedia.cs

```diff
@@ -360,5 +359,137 @@ private static PlaySourceInternal TranslatePlaySourceToInternal(PlaySource playS
             else
             { return null; }
         }
+
+        /// <summary>
+        /// Starts continuous Dtmf recognition.
+        /// </summary>
+        /// <param name="continuousDtmfOptions">Configuration attributes to start continuous Dtmf Recognition.</param>
+        /// <param name="cancellationToken"></param>
+        /// <returns>Returns an Http response 200 for success, or an http failure error code</returns>
+        public virtual Response StartContinuousDtmfRecognition(ContinuousDtmfRecognitionOptions continuousDtmfOptions,
```

Comment:
Giving that these options only include a required attribute, I wonder if we could just keep the method simple passing the target, instead of creating a new object for it.
Worse case, in the future as we add more optional attributes, we create an overload of the operation.
Reference: https://azure.github.io/azure-sdk/dotnet_introduction.html#dotnet-parameters

Reply:
thank you!
I have made the suggested changed, kindly review again.